src/openfl/display/Bitmap.hx

```diff
@@ -58,6 +60,18 @@ class Bitmap extends DisplayObject {
 		
 	}
 	
+	override function __setStageReference(stage:Stage) {
+		
+		this.stage = stage;
```

Comment:
Call `super.__setStageReference (stage);` just in case someone adds something to `DisplayObject`?

Reply:
The idea here is to save a virtual call. I think at this low level it might make sense and it's not like _someone_ can randomly add something to the base implementation.